### DISCUSSION TITLE
GraphClient.Users[id].Sponsors.Ref missing?

### DISCUSSION THREAD

**Discussion by stefanboerner** [microsoftgraph/msgraph-sdk-dotnet#2493] [Q&A]
I'd like to modify the "Sponsors" list of a user as described [here](https://learn.microsoft.com/en-us/graph/api/user-post-sponsors?view=graph-rest-1.0&tabs=http) but it seems that GraphClient.Users[id].Sponsors.**Ref** is missing in the microsoft.graph 5.52.0 library. Will this be added in an upcoming version?

**Comment by andrueastman** [comment]
Follow up via https://github.com/microsoftgraph/msgraph-sdk-dotnet/issues/2522
